### PR TITLE
New section about removing clients

### PR DESCRIPTION
**Pull request by mateiw**
# Description

New section about removing clients.

# Target branches

Which product version does this PR apply to?

- [ ] Manager-4.0
- [x] Manager-4.1
- [x] Manager-4.2
- [x] Uyuni

# Links

Fixes https://github.com/SUSE/spacewalk/issues/12071

### CODE REVIEW

**Comment by Loquacity** [comment]
I've done a copy edit. One note is that the client cfg is being restructured in other PRs, which will stomp on the nav in this one. Let's merge those first, then deal with the conflicts here.

**Review by Loquacity — CHANGES_REQUESTED** [review]
Blocking merging for now, until the other client cfg nav PRs are merged.

File: modules/client-configuration/pages/delete-clients.adoc
Comment (keichwa) [inline modules/client-configuration/pages/delete-clients.adoc:14]:
```suggestion
  If the client cannot be contacted, you can cancel the deletion, or delete the client without cleaning up the configuration files.
```
This sounds strange.  Is this actually a synchronous action?

File: modules/client-configuration/pages/delete-clients.adoc
Comment (keichwa) [inline modules/client-configuration/pages/delete-clients.adoc:17]: I wouldn't call this an alternative.

File: modules/client-configuration/pages/delete-clients.adoc
Comment (keichwa) [inline modules/client-configuration/pages/delete-clients.adoc:5]: Later on, you say that the server tries to do this, and that sounds as if the server is sometimes able to do this without further user action.

**Review by keichwa — APPROVED** [review]

File: modules/client-configuration/pages/delete-clients.adoc
Comment (Loquacity) [reply modules/client-configuration/pages/delete-clients.adoc:14]: "you are given the option to"?

File: modules/client-configuration/pages/delete-clients.adoc
Comment (Loquacity) [reply modules/client-configuration/pages/delete-clients.adoc:17]:
```suggestion
You can also delete multiple clients using the system set manager.
```

File: modules/client-configuration/pages/delete-clients.adoc
Comment (Loquacity) [reply modules/client-configuration/pages/delete-clients.adoc:5]: Heh, this is the problem with passive voice :wink:

Comment (Loquacity) [reply modules/client-configuration/pages/delete-clients.adoc:5]:
```suggestion
This procedure works for both traditional and Salt clients.
```

**Review by jcayouette — APPROVED** [review]
Looks good.

**Comment by Loquacity** [comment]
@mateiw Are you happy with these suggestions? If so, let's apply them and merge today.

**Comment by Loquacity** [comment]
> Blocking merging for now, until the other client cfg nav PRs are merged.

Other PRs are now merged, and the conflict in the nav here resolved, so this can be merged once we have a tick from @mateiw on the suggestions.

**Comment by Loquacity** [comment]
Suggestions that were not applied in this PR are now applied in https://github.com/uyuni-project/uyuni-docs/pull/609

**Pull request merged**
